person.rb

```diff
@@ -1,17 +1,18 @@
 require './nameable'
 
```

Comment:
- [ ] Please create `add_rental` method in person class that will accept date and book as parameters to create rentals and for the person assign self.